### ISSUE TITLE
Error When Trying to Run Samples:

### ISSUE THREAD

**Issue by N4t4ly4** [org-scn-design-studio-community/lumiradesignercommunityext#35]

**Comment by ncodernet** [comment]
Hello,
could you please be a little more specific? When does the issue appear? Where is the error message shown? What works, what not? ...

Thanks.

**Comment by bensteffen** [comment]
Hey,

I guess I have the same issue.

I have the following enviroment:
- Windows 10, 64bit
- Lumira Designer 2.1 SP01
-  jdk1.8.0_162
- Eclipse Oxygen.3a Release (4.7.3a)
- SDK sample pack is from 2015-04-30 including the sdk.xsd-file (I don't know where to get - if there is - a more recent one)

I strictly followed "Developer Guide: Component SDK" to configure eclipse to get the SDK sample "coloredbox" running. When I try run the project in eclipse following problems were detected during run-validation:

`com.sap.db.jdbc
    - Missing constraint: Import Package: javax.net; version="0.0.0"
    - Missing constraint: Import Package: javax.net.ssl; version="0.0.0"
    - Missing constraint: Import Package: javax.security.auth; version="0.0.0"
    - Missing constraint: Import Package: javax.security.auth.x500; version="0.0.0"
    - Missing constraint: Import Package: javax.sql; version="0.0.0"
    - Missing constraint: Import Package: org.ietf.jgss; version="0.0.0"
com.sap.ip.bi is missing
`
Any idea what's the problem?

**Comment by bensteffen** [comment]
It seems that you may ignore this error. But you have to use the newer sample pack (in my case 2017-09-20). With this version you can now see a "Custom Components" tab including the SDK-component in Lumira Designer.

**Comment by ncodernet** [comment]
Yes, you can ignore the warning. If you are annoyed, simply turn Validation off in your Launch Configuration:
![launch](https://user-images.githubusercontent.com/5783371/42160382-3c805ebc-7df7-11e8-8ae1-c449932000c2.png)
